simplq/src/styles/homePage.module.scss

```diff
@@ -157,55 +157,53 @@
   div {
     width: 2rem;
     height: 0.25rem;
-    $type: open;
-    @if $type == open {
+    background-color: #3a3768;
+    &[open] {
       background-color: #fff;
-    } @else {
-      background-color: #3a3768;
     }
     border-radius: 0.5rem;
     transform-origin: 1px;
     transition: all 0.3s linear;
 
     &:nth-child(1) {
-      $type: open;
-      @if $type == open {
+      transform: rotate(0);
+      &[open] {
         transform: rotate(45deg);
-      } @else {
-        transform: rotate(0);
       }
     }
     &:nth-child(2) {
-      $type: open;
-      @if $type == open {
+      transform: translateX(0);
+      opacity: 1;
+      &[open] {
         transform: translateX(100%);
         opacity: 0;
-      } @else {
-        transform: translateX(0);
-        opacity: 1;
       }
     }
     &:nth-child(3) {
-      $type: open;
-      @if $type == open {
+      transform: rotate(0);
+      &[open] {
         transform: rotate(-45deg);
-      } @else {
-        transform: rotate(0);
       }
     }
   }
 }
 
-.menu-list {
+.left-nav {
   list-style: none;
   display: flex;
   flex-flow: row nowrap;
   margin: 0;
   position: fixed;
   top: 1.2rem;
   li {
-    color: #fff;
-    padding: 0 2rem;
+    padding: 0 1.5rem;
+  }
+  li:last-child {
+    @media (min-width: 787px) {
```

Comment:
here and other places, can we switch to `rem`s if possible?

Reply:
You want screen size to be in rems? But the standard media queries are preferred to be written in px.
But if you want in rem, I will change it.

Reply:
I see. Let me read up, but for now, let this stay.